client/src/services/local-storage.js

```diff
@@ -0,0 +1,24 @@
+import config from "../config";
+
+export const loadStateFromLocalStorage = () => {
+        const serializedState = localStorage.getItem(config.stateKeyName);
+        if (serializedState === null) {
+            return undefined;
+        }
+        try{
+            return JSON.parse(serializedState);
+        }
+        catch(err) {
+            return undefined;
+        }
+};
+
+export const saveStateToLocalStorage = (state) => {
+    try{
+        const serializedState = JSON.stringify(state);
+        localStorage.setItem(config.stateKeyName, serializedState);
+    }
+    catch(err) {
```

Comment:
If you don't use the `err` prop so can just write `catch { ... }`
It's a convention when you just wan't to log an error, or completely ignore it

Reply:
```js
catch { }